assets/src/styles/blocks/PageHeader.scss

```diff
@@ -0,0 +1,107 @@
+.is-pattern-p4_page-header {
+  margin-inline-start: calc(((100vw - 100%) / 2) * -1);
+  padding: 0;
+  width: 100vw;
+
+  img.wp-block-cover__image-background {
+    height: inherit;
+    min-width: 360px;
+    max-height: 270px;
+  }
+
+  .wp-block-group {
+    padding-left: 24px;
+    padding-right: 24px;
+  }
+
+  .wp-block-group:first-child {
+    display: block;
+    margin-top: 233px;
+    max-width: 520px;
+  }
+
+  h1.has-background,
+  h2.has-background {
```

Comment:
Why do you also have `h2` in this selector? I don't see an `h2` tag in the pattern